web-services-app/src/test/java/edu/unc/lib/boxc/web/services/processing/IiifV3ManifestServiceTest.java

```diff
@@ -214,7 +239,7 @@ public void buildManifestViewableFileVideoTest() {
 
     @Test
     public void buildCanvasViewableFileTest() {
```

Comment:
Since it's new logic, it would be good to add a `buildCanvasNonViewableFileTest` to verify the new conditional in buildCanvas is working